Tests/DummyFile.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
```

Comment:
I saw an error in [StyleCI](https://github.styleci.io/analyses/zGAaOl) about this line but I can't see what's wrong exactly.
(the - and + line seems to be exactly the same).
Should I delete this line ?

Reply:
It looks like there's an extra empty line after `declare` line. Anyway, I don't see a reason to declare strict types in such file

Reply:
It's just creating by default with my phpstorm settings.
It's true  that in tests file it's not needed